difference.js

```diff
@@ -0,0 +1,5 @@
+function difference(array, values) {
```

Comment:
how will it work with `difference([1, 2, 3, 4, 5, 6], [3, 4, 5], [1,2,3], [5,6,3])`?

Reply:
fixed in 00dc018